_posts/en/newsletters/2021-03-10-newsletter.md

```diff
@@ -83,15 +82,15 @@ projects.
 
     - *Using time-based or height-based parameters:* the proposal
       describes the tradeoffs between setting its `start`, `timeout`,
-      and `minimum_activation` parameters using either times (based on
+      and `minimum_activation` parameters using either timestamps (based on
       the median of the previous 11 blocks) or block heights.  Using
-      times would result in the smallest and easiest to review patch to
+      timestamps would result in the smallest and easiest-to-review patch to
       Bitcoin Core.  Using heights would provide a bit more
       predictability, especially for miners, and would be compatible
       with other attempts using BIP8.
 
     - *Myopic:* there was [concern][russell concern] that the proposal
-      is too focused on the short term: "Speedy Trial fully prepares for
+      is too focused on the short term. Speedy Trial fully prepares for
       the (likely) case where miners activate taproot, but it does
       nothing to codify lessons learned from Segwit's failure to
       activate in a timely manner.  We have an opportunity with the
```

Comment:
Hm, maybe we should move away from first-person narration. One sec.

Reply:
Without the quotes, this seems a bit strong. Maybe it should be introduced with "Here's a summary of the argument:" followed by an indented block of text.

Reply:
Done (thanks!)